tests/testthat/test_indicator_native_range_year.R

```diff
@@ -0,0 +1,7 @@
+library(readr)
+data_input_checklist_indicators <- read_delim("https://raw.githubusercontent.com/trias-project/indicators/master/data/interim/data_input_checklist_indicators.tsv", 
+                                              "\t", escape_double = FALSE, trim_ws = TRUE)
+
+source("./R/indicator_native_range_year.r")
+
+countYearNativerange(data_input_checklist_indicators, jaartallen = c(1990:2019), type = "native_range", relative = FALSE)
```

Comment:
This is not a test. Testing plotly output is practically impossible or quite challenging, but you can test the data slot, and the fact that you get always a list and not something else... Writing tests is something which takes time, I know, but it helps making your code more robust and user friendly. For example, testing the input types,  See other tests functions in trias package to understand what I mean. But I don't blame you if you don't want to spend time on the tests, I will do it.

Reply:
> This is not a test.

![image](https://user-images.githubusercontent.com/16779320/90119063-608e7c80-dd59-11ea-9e8f-4b7b001c6074.png)

all jokes aside, I simply used this to test if the function worked.